src/lib/components/Instances/InstanceTableRow.svelte

```diff
@@ -127,17 +125,19 @@
 				<ArrowRight class="h-4" />
 			</button>
 		</td>
-		<td class="p-3">
-			{#if instance.label}
-				{#await metric}
-					Loading...
-				{:then metric}
-					{#if metric}
-						{(Math.round(metric.metric * 100) / 100).toFixed(2)}
-					{/if}
-				{/await}
-			{/if}
-		</td>
+		{#if project.metric_name !== null}
+			<td class="p-3">
+				{#if instance.label}
+					{#await metric}
```

Comment:
this isn't a promise anymore right? why await?

Reply:
whoopsies good catch!